### ISSUE TITLE
ubuntu 16.04.3 fatal error No such file or directory include ps4.h

### ISSUE THREAD

**Issue by hejran7** [idc/ps4-payload-sdk#6]
gcc -c -o build/MassStoreCommands.o source/MassStoreCommands.c -I/libPS4/include -I. -Iinclude -std=c11 -O2 -fno-builtin -nostartfiles -nostdlib -Wall -masm=intel -march=btver2 -mtune=btver2 -m64 -mabi=sysv -mcmodel=large
In file included from source/MassStoreCommands.c:51:0:
include/MassStoreCommands.h:41:20: fatal error: ps4.h: No such file or directory
    #include <ps4.h>
                    ^
compilation terminated.
Makefile:20: recipe for target 'build/MassStoreCommands.o' failed
make: *** [build/MassStoreCommands.o] Error 1

the only workaround if i copy the content from 
 /home/user/Desktop/SDK/libPS4/include
To 
/home/user/Desktop/SDK/libusbfatfs/include
not sure if am doing the right thing

**Comment by ZackBeal** [comment]
Getting the same error compiling on a CentOS7 host.

**Comment by idc** [comment]
Set `PS4SDK` environment variable.

**Comment by hejran7** [comment]
thanks for your replay idc, here is my setting 

PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
PS4SDK="/home/user/Desktop/PS4_SDK"

the PS4_SDK on the desktop still getting the same error, not so sure but i think its ubuntu on windows 7 issue

**Comment by hejran7** [comment]
issue solved 
if u getting the same error am getting u have to add a new environment variable in Ubuntu permanently
Open a terminal or by pressing Ctrl Alt T 
sudo -H gedit /etc/environment
a  text file will opened so u need to edit 
in my case the PS4_SDK is on my desktop so i add  
PS4SDK="/home/yourusername/Desktop/PS4_SDK"
save the changes and close ur text file
restart Ubuntu hope it will works for u just how it works for me

**Comment by idc** [comment]
Yes, sorry, my initial reply suggested `PS4_SDK` which was wrong, when it should have been `PS4SDK`.

**Comment by rezaadi0105** [comment]
thanks working
